TESTS/netsocket/udp/udpsocket_sendto_invalid.cpp

```diff
@@ -32,7 +32,14 @@ void UDPSOCKET_SENDTO_INVALID()
     TEST_ASSERT(sock.sendto(NULL, 9, NULL, 0) < 0);
     TEST_ASSERT(sock.sendto("", 9, NULL, 0) < 0);
     TEST_ASSERT(sock.sendto("", 0, NULL, 0) < 0);
-    TEST_ASSERT_EQUAL(0, sock.sendto(MBED_CONF_APP_ECHO_SERVER_ADDR, 9, NULL, 0));
+
+    nsapi_error_t result = sock.sendto(MBED_CONF_APP_ECHO_SERVER_ADDR, 9, NULL, 0);
+    if (result == NSAPI_ERROR_UNSUPPORTED) {
+        TEST_IGNORE_MESSAGE("sendto() sending empty packet not supported");
```

Comment:
This ignores the whole testcase.
Maybe just
```
if (result != NSAPI_ERROR_UNSUPPORTED) {
    TEST_ASSERT_EQUAL(0, result);
}
```

Reply:
Fixed